website/docs/core-concepts/posts.md

```diff
@@ -0,0 +1,49 @@
+---
+sidebar_position: 2
+---
+
+# Posts 
+
+Posts are articles or stories which may include images or other illustrations.
+New Post can be created from the Posts menu by clicking on the New Post button on the right side.
+
+The user can start writing a post using an editor which is a rich text editor. It makes embedding images, links, or any related content very easy.
+
+The user can directly publish or can save it as a draft. The user can also mark the post as ready to Publish.
+ 
+Each post created has status from any of the following.
+- Draft
+- Published
+- Ready to Publish
+
+To save as draft click on the Save button on the right and to save as Ready to Publish, before clicking on the Save button make sure Ready to Publish is checked.
+
+To publish the Post click on the Publish button. Only the user who is either admin or an editor can publish the Post.
+
+Once the post is created, templates can be created from the post which can be used for future posts. Templates can be created by clicking on the Create Template button.
+New posts can be created from templates by clicking on any of the templates available from the Templates section which is at the top of the Post page
+
+The user can add information about the post by clicking on the Settings button.
+
+- Featured Image - Image to be set as thumbnail for the post
```

Comment:
not just thumbnail, a featured image is more than that. check this on wp for eg: https://wordpress.com/support/featured-images